tool.ts

```diff
@@ -234,11 +265,12 @@ function _completeToolPath(tool:string, version: string, arch?: string): void {
 export async function cacheDir(sourceDir: string,
                                tool: string,
                                version: string,
-                               arch?: string) {
-    tl.debug('caching directory');
+                               arch?: string): Promise<string> {
+    version = semver.clean(version);
     arch = arch || os.arch();
+    console.log(`Caching tool: ${tool} ${version} ${arch}`);
```

Comment:
Changed to console.log since the cp can take time. We could potentially improve copy time by optionally hardlinking files (does not require elevated, irrc).